Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -162,7 +162,9 @@ private extension MoyaProvider {
                     self.cancelCompletion(completion, target: target)
                     return
                 }
-                cancellable.innerCancellable = self.sendAlamofireRequest(alamoRequest, target: target, callbackQueue: callbackQueue, progress: progress, completion: completion)
+                let validationCodes = target.validationType.statusCodes
+                let validated = validationCodes.isEmpty ? alamoRequest : alamoRequest.validate(statusCode: validationCodes)
```

Comment:
`validatedRequest`?

Reply:
Ugh, that is what I had originally. I knew I should've left it 😆